dask/dataframe/io/parquet.py

```diff
@@ -346,46 +538,33 @@ def to_parquet(df, path, compression=None, write_index=None, has_nulls=True,
     path : string
         Destination directory for data.  Prepend with protocol like ``s3://``
         or ``hdfs://`` for remote data.
+    engine : {'auto', 'fastparquet', 'pyarrow'}, default 'auto'
+        Parquet library to use. If only one library is installed, it will use
+        that one; if both, it will use 'fastparquet'
     compression : string or dict
         Either a string like "SNAPPY" or a dictionary mapping column names to
         compressors like ``{"name": "GZIP", "values": "SNAPPY"}``
     write_index : boolean
         Whether or not to write the index.  Defaults to True *if* divisions are
         known.
-    has_nulls : bool, list or 'infer'
-        Specifies whether to write NULLs information for columns. If bools,
-        apply to all columns, if list, use for only the named columns, if
-        'infer', use only for columns which don't have a sentinel NULL marker
-        (currently object columns only).
-    fixed_text : dict {col: int}
-        For column types that are written as bytes (bytes, utf8 strings, or
-        json and bson-encoded objects), if a column is included here, the
-        data will be written in fixed-length format, which should be faster
-        but can potentially result in truncation.
-    object_encoding : dict {col: bytes|utf8|json|bson} or str
-        For object columns, specify how to encode to bytes. If a str, same
-        encoding is applied to all object columns.
-    storage_options : dict
-        Key/value pairs to be passed on to the file-system backend, if any.
-    append: bool (False)
+    append : bool (False)
         If False, construct data-set from scratch; if True, add new
         row-group(s) to existing data-set. In the latter case, the data-set
         must exist, and the schema must match the input data.
-    ignore_divisions: bool (False)
+    ignore_divisions : bool (False)
         If False raises error when previous divisions overlap with the new
         appended divisions. Ignored if append=False.
-    partition_on: list
+    partition_on : list
         Construct directory-based partitioning by splitting on these fields'
         values. Each dask partition will result in one or more datafiles,
         there will be no global groupby.
-    times: 'int64' (default), or 'int96':
```

Comment:
`pyarrow` also supports this mode, this can be toggled with `use_deprecated_int96_timestamps` on `pq.write_tablee`.

Reply:
As is, this matches the pandas behavior, just forwarding the extra kwargs to the backend (you can pass `times` to `fastparquet` and `use_deprecated_int96_timestamps` to `pyarrow`).

Personally I'd like to unite on one of these names and deprecate the other (I prefer the long `use_deprecated_int96_timestamps`). I'd rather not implicitly unite on them though until pandas settles on the same behavior (we like to match pandas as much as possible).